### PR TITLE
Brush up some FFoF actors

### PR DESCRIPTION
**Pull request by simonward**

### CODE REVIEW

File: packs/one-shot-bestiary/darius.json [hunk 1]
```diff
@@ -455,96 +455,80 @@
                     {
                         "domain": "all",
                         "key": "RollOption",
-                        "label": "Darius has an active mutagen effect",
+                        "label": "PF2E.NPCAbility.Darius.ActiveMutagen",
```
Comment (TikaelSol) [inline packs/one-shot-bestiary/darius.json:458]: I don't know that we even need a label, this ability is kind of weird I think.

Comment (simonward) [reply packs/one-shot-bestiary/darius.json:458]: It is pretty weird. I thought about not having a custom label, but I'm not sure which state "Metabolize Mutagen" describes. Suggestions?

Comment (TikaelSol) [reply packs/one-shot-bestiary/darius.json:458]: I think we may want to just redo a lot of his automation. Maybe a custom effect for the mutagen for him to get him the right modifiers and all that.

Comment (simonward) [reply packs/one-shot-bestiary/darius.json]: Moved to an effect.

**Pull request merged**